_content/tour/por/algorithms-data.article

```diff
@@ -42,16 +42,16 @@ This sample program implements a basic hash table.
 
 .play algorithms/data/hash_map.go
 
-* Linked List
+* Lista Encadeada
 
-This sample program implements a basic double linked list.
+Este programa de exemplo implementa uma lista duplamente encadeada básica.
 
-- See more at [[https://en.wikipedia.org/wiki/Linked_list][https://en.wikipedia.org/wiki/Linked_list]]
+- Veja mais em [[https://en.wikipedia.org/wiki/Linked_list][https://en.wikipedia.org/wiki/Linked_list]]
 
-*Diagram*
+*Diagrama*
 
-    With a linked list, values are tied together in different
-    order through the use of pointers.
+    Com uma lista encadeada, os valores são vinculados entre si
+    em ordem diferente, por meio do uso de ponteiros.
```

Comment:
I didn't understand this part: "in different order", maybe someone can help.

Reply:
The values are not stored in contiguous block of memory, like an array. Linked lists store values at different memory locations, so they are linked together througnt these pointers.

Reply:
Do you think "Com uma lista encadeada, os valores ficam em diferentes locais da memória e são vinculados por meio do uso de ponteiros." would fit better?

Reply:
Nice one @rschio